v63003/u_inref/gtm8787.csh

```diff
@@ -14,40 +14,35 @@
 # Tests that Mupip Journal -Extract=-stdout handles its termination appropriately
 #
 
-# Creating database,journal file
-setenv gtm_test_db_format "NO_CHANGE"
-source $gtm_tst/com/gtm_test_setbgaccess.csh
-source $gtm_tst/com/gtm_test_setbeforeimage.csh
+echo "# Creating database,journal file"
 $gtm_tst/com/dbcreate.csh mumps 1>>& create.out
 if ($status) then
 	echo "DB Create Failed, Output Below"
 	cat create.out
 endif
-$MUPIP Set -Region Default -Journal=enable,on,before,file=mumps.mjl >>& jnlcreate.out
+$MUPIP Set -Region Default -Journal=enable,on,nobefore,file=mumps.mjl >>& jnlcreate.out
```

Comment:
Add a comment that we use nobefore so this test can work with BG or MM access method (randomly chosen by test framework).